.github/renovate.json5

```diff
@@ -0,0 +1,42 @@
+{
+	$schema: "https://docs.renovatebot.com/renovate-schema.json",
+	extends: [
+		"config:recommended",
+		":approveMajorUpdates",
+		":semanticCommitScopeDisabled",
+	],
+	ignorePresets: [":semanticPrefixFixDepsChoreOthers"],
+	labels: ["dependencies"],
+
+	// Wait well over npm's three day window for any new package as a precaution against malicious publishes
+	// https://docs.npmjs.com/policies/unpublish/#packages-published-less-than-72-hours-ago
+	minimumReleaseAge: "7 days",
+
+	packageRules: [
+		{
+			description: "Use the deps:actions label for github-action manager updates (this means Renovate's github-action manager).",
+			addLabels: ["deps:actions"],
+			matchManagers: ["github-actions"],
+		},
+		{
+			description: "Use the deps:npm label for npm manager packages (this means Renovate's npm manager).",
+			addLabels: ["deps:npm"],
+			matchManagers: ["npm"],
+		},
+		{
+			description: "Group Babel packages into a single PR.",
+			groupName: "babel",
+			matchPackagePrefixes: ["@babel", "babel-"],
+		},
```

Comment:
This can be removed as we aren't using Babel in this repo

Reply:
removed `@wdio` as I didn't see it in the repo either